src/WinUI/ArcGIS.WinUI.Viewer/ArcGIS.WinUI.Viewer.csproj

```diff
@@ -79,10 +79,6 @@
     </Compile>
   </ItemGroup>
 
-  <ItemGroup>
```

Comment:
Removing this, as sketch-on-map samples has been replaced with geometry editor sample.